helm/messenger/templates/deployment.yaml

```diff
@@ -82,7 +82,24 @@ spec:
           volumeMounts:
             - name: src
               mountPath: /target/
-    {{- end }}
+        {{- end }}
+        - name: init-conf
+          image: mikefarah/yq:4
+          command: ["sh"]
+          args:
+            - '-c'
+            - >
+              set -ex;
+              touch /tmp/.yaml && printf {{ toYaml $.Values | quote }} >> /tmp/.yaml;
+              for e in {{ join " " $.Values.ingress.hosts | trimPrefix "[" | trimSuffix "]" }}; do yq '. * {"conf":."'$e'" // {}} | withEntries(select(.key == "conf")) | .conf as $root | {} | . = $root' /tmp/.yaml > /tmp/conf/$e.yaml; done;
+              rm -rf /tmp/.yaml;
+          volumeMounts:
+          - name: init
+            mountPath: /tmp/conf
+          - name: conf
+            subPath: conf.toml
+            mountPath: /var/run/conf.toml
+            readOnly: true
```

Comment:
Oopsie, I guess this mount is unnecessary, since the `init-conf` doesn't use it. It reads the whole `.Values` to form separate `.yaml` files for domains.